environment.yml

```diff
@@ -15,11 +15,10 @@ dependencies:
   - notebook
   - joblib
   - toml
-  - protobuf  # for protozfits
-  - zeromq    # for protozfits
+  - protozfits=2.0
```

Comment:
Not repeating it here, will result in installing it from pypi, since it will get the rquirememt from setup.py, not the comda environment.

This goes by the way for pretty much all of the dependencies.

Reply:
Since there is no `conda-forge` package for lstchain (I could do it if there is interest), conda does not handle dependencies of lstchain. This is the reason why this environment file exists, to install the dependencies (and useful packages I might add, we have more in this env than the strict dependencies of lstchain I think) that are available through conda from conda before running `pip install lstchain`.

If we would have  a conda package for lstchain (e.g. via conda-forge) then `conda install conda-forge::lstchain` would also install all lstchain dependencies via conda (including ctapipe_io_lst and thus protozfits).

But that requires that all lstchain dependencies (including ctapipe_io_lst) are available via conda(-forge).

Reply:
Which at the moment is not true for at least `ctapipe_io_lst` and `pyirf` which would have to come first.

I can try to do it this week. Especially for python-only packages, it is not hard.